### DISCUSSION TITLE
Add notification count badge to Notifications sidebar component

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3059] [Feature Requests]
We should have an indicator for any unread activity/notification items in the sidebar:

![Screen Shot 2020-09-30 at 5 43 35 PM](https://user-images.githubusercontent.com/522079/94743171-f18ad880-0344-11eb-8aa4-361b71fc4010.png)

![Screen Shot 2020-09-30 at 5 46 30 PM](https://user-images.githubusercontent.com/522079/94743177-f485c900-0344-11eb-821e-cc85a85376d6.png)

**Comment by danieljwestman** [comment]
Yes please - would’ve been awesome!

**Comment by benhaynes** [comment]
@rijkvanzanten — will this require a "seen by user" junction table? Another option would be to save the timestamp the user checked notifications, and consider anything after that unread.

**Comment by rijkvanzanten** [comment]
This exists now!
